src/execution/values.ts

```diff
@@ -272,7 +276,7 @@ export function experimentalGetArgumentValues(
         valueNode,
         argType,
         (error, path) => {
-          error.message = `Argument "${argDef.name}" has invalid value${printPathArray(
+          error.message = `Argument "${isArgument(argDef) ? argDef : argDef.name}" has invalid value${printPathArray(
```

Comment:
Yes, it could be a variable signature, which is "a kind of argument definition", but confusing. We should probably rename `argDef`, but I would guess that should be in a separate PR.

Reply:
646d600